less_slow.py

```diff
@@ -553,6 +553,24 @@ def kernel():
     assert result == 3.0
 
 
+@dataclass
```

Comment:
It may be a wise idea to merge slots and uv first and reconsider the rest in a separate PR 🤗

Reply:
Agree! I'll open a separate PR tomorrow with just those two changes.